www/scripts/main.js

```diff
@@ -229,6 +230,11 @@ function setLastRoomName(roomName) {
 
 /* Helpers */
 
+function displayMessage(recipient) {
```

Comment:
*sits in corner with arms folded refusing to celebrate non statically typed languages*